functions/ocr/ocr-translate-text/src/main/java/functions/OcrTranslateText.java

```diff
@@ -36,10 +36,11 @@
 import java.util.logging.Logger;
 
 public class OcrTranslateText implements BackgroundFunction<PubSubMessage> {
+  private static final Logger logger = Logger.getLogger(OcrTranslateText.class.getName());
```

Comment:
I think this change was already handled [here](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/2805)?

(Maybe we should revert this line to reduce the risk of merge conflicts.)

Reply:
I had to merge manually, since this declaration needs to be before the calls to `getenv` (since those can issue log messages).